x-pack/plugins/session_view/public/components/session_view/index.tsx

```diff
@@ -239,6 +241,10 @@ export const SessionView = ({
     onSearchIndexChange(0);
   }, [onSearchIndexChange, searchResults]);
 
+  useEffect(() => {
```

Comment:
to avoid having empty `sessionViewId` during the first render we should be able to do 
```
const sessionViewId = useMemo(() => `session-view-uuid-${uuidv4()}`, []);
```
it is generated right on the first render then and is preserved between renders

alternative is 
```
const [sessionViewId] = useState(() => `session-view-uuid-${uuidv4()}`);
```
then you don't need the `useEffect`